### PR TITLE
Host Plans: Dynamically fetch plans

### PR DESCRIPTION
**Pull request by kewitz**

### CODE REVIEW

**Comment by znarf** [comment]
@leonardohof The descriptions that were in the previous version can't be done with `collective.description`. Seems `collective.longDescription` should be used because it's allowing HTML.

File: components/edit-collective/EditHostSettings.js [hunk 1]
```diff
@@ -38,158 +41,79 @@ const PlanPrice = styled.p`
...
-            <strong>Host Dashboard</strong>: {collective.plan.hostDashboard ? 'Yes' : 'No'}
-          </li>
-        </ul>
-      </div>
-    );
-  }
-}
+const EditHostSettings = props => {
+  const { collective } = props;
+  const { data: opencollective } = useQuery(getCollectiveQuery, {
```
Comment (znarf) [inline components/edit-collective/EditHostSettings.js]: Can you use a custom query instead of the `getCollectiveQuery` ?

Comment (znarf) [reply components/edit-collective/EditHostSettings.js]: And make sure we query `longDescription` on Tier.

**Comment by znarf** [comment]
I already added support for `longDescription` on the branch but not the GraphQL query to read it.

File: components/edit-collective/EditHostSettings.js (hunk 1 above)
Comment (kewitz) [reply components/edit-collective/EditHostSettings.js]: Sure, I'll write a custom query in this file.

**Pull request merged**